packages/docusaurus-theme-classic/src/theme/DocSidebarItem/index.tsx

```diff
@@ -139,7 +139,13 @@ function DocSidebarItemCategory({
         },
         className,
       )}>
-      <div className="menu__list-item-collapsible">
+      <div
+        className={clsx(
+          ThemeClassNames.docs.docSidebarItemCategoryLink,
+          ThemeClassNames.docs.docSidebarItemCategoryLinkLevel(level),
+          'menu__list-item-collapsible',
+          linkClassName,
```

Comment:
The user wanted the className to be applied to the `<li>` item, so may not exactly solve the user problem